cleanlab/dataset.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+
+"""Dataset Module
+Supports dataset-level and class-level automated quality, including finding which classes should
+be merged with other classes in your dataset, which classes should be removed,
+and which classes have the best label accuracy, and which have the worst labels.
+"""
+
+import numpy as np
+from count import estimate_joint, compute_confident_joint
+
+
+def find_similar_classes(
+    joint=None,
+    asymmetric=True,
+    *,
+    labels=None,
+    pred_probs=None,
```

Comment:
I find this argument order a bit peculiar.
Consider having the arg order be like this match the other parts of package:
```
labels=None,
pred_probs=None,
*,
joint=None,
confident_joint=None,
asymmetric=True,
```

In particular, I'd think about it like this: 100% of cleanlab users will know what `labels` and `pred_probs` are. But only say 50% of users will know what `joint` and `confident_joint` are.